docs/source/_includes/create-genesis-block.inc

```diff
@@ -0,0 +1,215 @@
+The first node in a new Sawtooth network must create the `genesis block` (the
+first block on the distributed ledger). When the other nodes join the network,
+they use the on-chain settings that were specified in the genesis block.
+
+The genesis block specifies the consensus algorithm and the keys for nodes (or
+users) who are authorized to change configuration settings. For PBFT, the
+genesis block also includes the keys for the other nodes in the initial network.
+
+.. important::
+
+   Use this procedure **only** for the first validator node on a Sawtooth
+   network. Skip this procedure for a node that will join an existing network.
+
+1. Ensure that the required user and validator keys exist on this node:
+
+   .. code-block:: console
+
+      $ ls ~/.sawtooth/keys/
+      {yourname}.priv    {yourname}.pub
+
+      $ ls /etc/sawtooth/keys/
+      validator.priv   validator.pub
+
+   If these key files do not exist, create them as described in the previous
+   step.
+
+#. Become the ``sawtooth`` user. Otherwise, you must preface each command with
+   ``sudo -u sawtooth``.
+
+   .. code-block:: console
+
+      $ sudo -u sawtooth -s
+      [sawtooth@system]$ 
+
+   In the following commands, the prompt ``[sawtooth@system]`` shows the
+   commands that must be executed as the ``sawtooth`` user.
+
+#. Change to a writable directory such as ``/tmp``.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ cd /tmp
+
+#. Create a batch with a settings proposal for the genesis block.
+
+   .. code-block:: console
+
+      [sawtooth@system]$ sawset genesis \
+      --key /etc/sawtooth/keys/validator.priv \
+      -o config-genesis.batch
+
+   This command authorizes this key (the validator key on this node) to change
+   Sawtooth settings. You must use the same key for the following commands in
+   this procedure. Also, any later commands to change on-chain Sawtooth settings
+   must specify this key.
+
+   .. note::
+
+      This procedure uses the same validator key for all commands that require a
+      key. In theory, some of these commands could use a different key, but
+      configuring multiple keys is a complicated process. Describing how to use
+      multiple keys to create the genesis block is outside the scope of this
+      guide.
+
+#. Create a batch to initialize the consensus settings.
+
+   * For PBFT:
+
+     .. code-block:: console
+
+        [sawtooth@system]$ sawset proposal create \
+        --key /etc/sawtooth/keys/validator.priv \
+        -o config-consensus.batch \
+        sawtooth.consensus.algorithm.name=pbft \
+        sawtooth.consensus.algorithm.version=VERSION \
+        sawtooth.consensus.pbft.members=[VAL1KEY, VAL2KEY, VAL3KEY]
+
+     Replace ``VERSION`` with the PBFT version number as listed in the file
+     ``sawtooth-pbft/Cargo.toml`` (see the ``[package]`` section).
+
+     Replace ``VAL1KEY, VAL2KEY, VAL3KEY`` with the validator public
+     keys of the other nodes. This information is in the file
```

Comment:
This should say "all nodes"; this list must include this node as well as the other nodes. The example list should also show 4 keys since 4 is the minimum number of members.

Reply:
Fixed.